From 05e9e5ed36208ad7e1d4062305e6d43862bbac40 Mon Sep 17 00:00:00 2001
From: Rico Tzschichholz <ricotz@ubuntu.com>
Date: Sun, 15 Jan 2023 09:22:29 +0100
Subject: [PATCH] sd/qa: Increase range for DPI dependent check of BoundRect

[_RUN_____] SdOOXMLExportTest3::testTdf125573_FontWorkScaleX
./sd/qa/unit/export-tests-ooxml3.cxx:751:SdOOXMLExportTest3::testTdf125573_FontWorkScaleX
less assertion failed
- Expected less than: 50
- Actual  : 57

Change-Id: I32edc94b5060e05b382c478770e9bd3ba444dadd
---
 sd/qa/unit/export-tests-ooxml3.cxx | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sd/qa/unit/export-tests-ooxml3.cxx b/sd/qa/unit/export-tests-ooxml3.cxx
index 309709317821..240579639d47 100644
--- a/sd/qa/unit/export-tests-ooxml3.cxx
+++ b/sd/qa/unit/export-tests-ooxml3.cxx
@@ -748,14 +748,14 @@ void SdOOXMLExportTest3::testTdf125573_FontWorkScaleX()
     awt::Rectangle aBoundRectArch;
     xShapeArchProps->getPropertyValue(UNO_NAME_MISC_OBJ_BOUNDRECT) >>= aBoundRectArch;
     // BoundRect is DPI dependent, thus allow some range.
-    CPPUNIT_ASSERT_LESS(sal_Int32(50), std::abs(aBoundRectArch.Width - 13038));
+    CPPUNIT_ASSERT_LESS(sal_Int32(60), std::abs(aBoundRectArch.Width - 13038));
 
     // Error was, that text in shapes of category "Warp" was not scaled to the path.
     uno::Reference<beans::XPropertySet> xShapeWaveProps(getShapeFromPage(0, 1));
     awt::Rectangle aBoundRectWave;
     xShapeWaveProps->getPropertyValue(UNO_NAME_MISC_OBJ_BOUNDRECT) >>= aBoundRectWave;
     // BoundRect is DPI dependent, thus allow some range.
-    CPPUNIT_ASSERT_LESS(sal_Int32(50), std::abs(aBoundRectWave.Width - 11576));
+    CPPUNIT_ASSERT_LESS(sal_Int32(60), std::abs(aBoundRectWave.Width - 11576));
 }
 
 void SdOOXMLExportTest3::testTdf99497_keepAppearanceOfCircleKind()
-- 
2.38.1

